### PR TITLE
Feature merge to develop edited

### PR DESCRIPTION
**Pull request by manishmowall**
Pull request comment To feature to merge to develop edited

### CODE REVIEW

**Comment by nitishm-webonise** [comment]
first single comment

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+*~
+out
+*.iml
```
Comment (manishmowall) [inline .gitignore:3]: first review comment

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+.idea
```
Comment (manishmowall) [inline .gitignore:4]: second review comment edited

**Review by manishmowall — COMMENTED** [review]
review submit 1 edited

File: .gitignore (hunk 2 above)
Comment (nitishm-webonise) [inline .gitignore]: adf

File: src/main/java/org/webonise/polygon/Application.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
+package org.webonise.polygon;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Level;
```
Comment (manishmowall) [inline src/main/java/org/webonise/polygon/Application.java:5]: second submit comment to review to 2 edited

**Review by manishmowall — COMMENTED** [review]
second submit edited edited

File: src/main/java/org/webonise/polygon/Application.java [hunk 4]
```diff
@@ -0,0 +1,127 @@
+package org.webonise.polygon;
+
+import java.util.InputMismatchException;
```
Comment (manishmowall) [inline src/main/java/org/webonise/polygon/Application.java:3]: single comment edited

File: src/main/java/org/webonise/polygon/Application.java [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+        logger = Logger.getLogger(Application.class.getName());
+        polygonCalculator = new PolygonCalculator();
+        input = new java.util.Scanner(System.in);
+    }
+
+    public void start() {
+
+        try {
+            startApplication();
+        } catch (IllegalArgumentException | InputMismatchException illegalArgumentException) {
```
Comment (manishmowall) [inline src/main/java/org/webonise/polygon/Application.java:25]: commented

**Pull request merged**